### PR TITLE
App Design - Bar Graph & Filters

### LINKED ISSUE #30: App Design - Bar Graph
-> Bar graph by city, colored by state
-> Legend of states
->  Tool tip with descriptive information
->  Filters to work along with the map

### PR DESCRIPTION
**Pull request by zgarciaj**
**NOT READY FOR MERGER**

Feel free to test out the first few filters on the Bar Graph.

Waiting for layout lecture to complete filters and application on bar graph.

### CODE REVIEW

**Comment by zgarciaj** [comment]
Example

File: src/app.py [hunk 1]
```diff
@@ -1,24 +1,167 @@
...
-    html.H3("This dashboard is for UBC MDS Program DSCI 532"),
-    html.H4("Authors:"),
-    html.Ul([
-        html.Li("Ruocong Sun"),
-        html.Li("Chun Li"),
-        html.Li("Jake Barnabe"),
-        html.Li("Zeily Garcia")
-    ])
+# Load and prepare the dataset
+df = pd.read_csv('../data/processed/processed_df.csv')
```
Comment (lichunubc) [inline src/app.py]: I think we might have to remove the `../` in front of `data` to make it compatible on render.com.
In short, this line should be:
df = pd.read_csv('../data/processed/processed_df.csv')
The `../data` is required for testing things locally, but when deployed, the default working directory is the root of our repo which means we do not to go one folder down first

Comment (zgarciaj) [reply src/app.py]: You're absolutely right @lichunubc 
Will commit that change asap, thank you!

**Comment by lichunubc** [comment]
Looks fantastic

**Pull request merged**